menu_student.html

```diff
@@ -0,0 +1,18 @@
+{% extends 'base_student.html' %}
+{% block body %}
+
+<table><tr><td width=150px; max-width=150px; valign='top'; style="border-right: 1px solid black; padding: 5px;">
+  <h3>Step 1</h3>
+  <li><a href="/preregistration?{{ session_query }}">View My Catalog</a>
+  <li><a href="/catalog_full?{{ session_query }}">View Full Catalog</a>
+  <h3>Step 2</h3>
+  <li><a href="/schedule?{{ session_query }}">Register For Classes</a>
+  <li><a href="/preferences?{{ session_query }}">Order By Preference</a>
```

Comment:
You might be able to make this menu work with impersonation if you add a new parameter for the student to impersonate:

{% if student_to_impersonate %}
  href="/preferences?{{ session_query }}&student={{ student_to_impersonate }}"
{% endif %}